invenio_communities/views/communities.py

```diff
@@ -98,10 +98,32 @@ def communities_settings(community=None, logo=None, pid_value=None):
         },
         # Pass permissions so we can disable partially UI components
         # e.g Settings tab
-        permissions=community.has_permissions_to(['update']),
+        permissions=permissions(community),
         active_menu_tab="settings"
     )
 
+@require_create_permission
```

Comment:
I don't think this is correct: the tab should probably displayed when you are authenticated